src/helpers.tsx

```diff
@@ -0,0 +1,21 @@
+import { Todo } from './types/Todo';
+
+export const filterTodos = (
+  todos: Todo[],
+  filter: string,
+  query: string,
+) => {
+  return todos.filter(todo => {
+    switch (filter) {
+      case 'completed':
```

Comment:
Let's use you enum values here
```suggestion
      case Filters.Completed:
```